insights/combiners/package_provides_httpd.py

```diff
@@ -33,6 +34,10 @@ class PackageProvidesHttpdAll(LegacyItemAccess):
     """
 
     def __init__(self, package_provides_httpd):
+        deprecated(
```

Comment:
@xiangce since this method is not included in the documentation, could you please add info in the docstring about deprecation of this combiner, thanks.

Reply:
I will add the corresponding `warning` to the docstring in the next commit